spec/models/rules_of_origin/sidebar_section_spec.rb

```diff
@@ -12,8 +12,8 @@
   it { is_expected.to respond_to :name }
   it { is_expected.to respond_to :steps }
 
-  describe '#active?' do
-    subject { instance.active? }
+  describe '#current?' do
+    subject { instance.current? }
```

Comment:
```suggestion
    subject(:current?) { instance.current? }
```

I'm surprised rubocop doesn't flap at this :)